packages/synapse-react-client/src/components/FeaturedDataTabs/FeaturedDataTabs.test.tsx

```diff
@@ -16,7 +16,7 @@ const renderComponent = (props: FeaturedDataTabsProps) => {
 describe('FeaturedDataTabs tests', () => {
   const props: FeaturedDataTabsProps = {
     rgbIndex: 0,
-    sql: 'select * from syn11346063',
+    sql: 'select * from syn50000',
```

Comment:
replaced test references of syn11346063 to the mock file view syn50000 due to a flaky test that was failing when it attempted to get the file view entity